src/main.rs

```diff
@@ -20,11 +33,128 @@ fn main() {
             );
             (x, y)
         })
-        .collect();
+        .collect()
+    } else {
+        Vec::new()
+    };
 
-    let mut grid = Grid::new(10, 10);
+    let mut grid = Grid::new(20, 20);
 
-    //let points = vec![(5, 4), (4, 4), (3, 4), (4, 3), (4, 5)];
     grid.seed(points);
-    grid.start();
+
+    let mut generation = 0;
+    let mut auto_advance = false;
+    let mut time_sleep = 1.0;
+    let mut acc_time = 0.;
+    let mut chosen_cell_x = 0;
+    let mut chosen_cell_y = 0;
+
+    loop {
+        clear_background(WHITE);
+        let text = "Generation Number ".to_string() + generation.to_string().as_str();
```

Comment:
```suggestion
          let text = format!("Generation Number {}", generation);
```
I think this looks nicer